crates/viewer/re_renderer/src/video/decoder/native_decoder.rs

```diff
@@ -129,6 +129,22 @@ fn copy_video_frame_to_texture(
     frame: &Frame,
     texture: &wgpu::Texture,
 ) -> Result<(), DecodingError> {
+    let format = match frame.format {
+        re_video::PixelFormat::Rgb8Unorm => {
+            return copy_video_frame_to_texture(
+                queue,
+                &Frame {
+                    data: crate::pad_rgb_to_rgba(&frame.data, 255_u8),
```

Comment:
I'd be a lot happier if this goes through texture manager's format conversions that landed on main now because that's what is going to happen with AV1 output now

Reply:
Yeah, that's the goal right?